### PR TITLE
`FormatCBFMiniEigerChessID7B2`

### PR DESCRIPTION
**Pull request by dagewa**
The rotation axis at CHESS ID7B2 is antiparallel to the `fast` direction, as spotted during work on https://github.com/dials/dials/issues/2464. This PR adds specific support for that beamline.

### CODE REVIEW

File: newsfragments/649.feature [hunk 1]
```diff
@@ -0,0 +1 @@
+Add CBFMini support for the EIGER2 16M detector at CHESS beamline ID7B2, with an inverted rotation axis.
```
Comment (graeme-winter) [inline newsfragments/649.feature:1]: Defer to @ndevenish for news formatting

File: src/dxtbx/format/FormatCBFMiniEigerChessID7B2.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+        header = FormatCBFMiniEiger.get_cbf_header(image_file)
+        for record in header.split("\n"):
+            if "Detector: Dectris EIGER2 Si 16M, S/N E-32-0123" in record:
+                return True
+
+        return False
+
+    def _goniometer(self):
+        """Return a model for a simple single-axis goniometer. This should
```
Comment (graeme-winter) [inline src/dxtbx/format/FormatCBFMiniEigerChessID7B2.py:30]: Suggest comment not useful?

File: src/dxtbx/format/FormatCBFMiniEigerChessID7B2.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    installed at CHESS ID7B2, which has an inverted goniometer axis."""
+
+    @staticmethod
+    def understand(image_file):
+        """Check to see if this looks like an Eiger mini CBF format image,
+        i.e. we can make sense of it."""
+
+        header = FormatCBFMiniEiger.get_cbf_header(image_file)
+        for record in header.split("\n"):
+            if "Detector: Dectris EIGER2 Si 16M, S/N E-32-0123" in record:
```
Comment (graeme-winter) [inline src/dxtbx/format/FormatCBFMiniEigerChessID7B2.py:24]: Suggest 

```
if "# Detector: ...
```

in record to make it clear to the reader that we are looking in the `#` demarked metadata island in the header

File: src/dxtbx/format/FormatCBFMiniEigerChessID7B2.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
+"""
```
Comment (graeme-winter) [inline src/dxtbx/format/FormatCBFMiniEigerChessID7B2.py:1]: Duplicated comment not useful?

**Review by graeme-winter — APPROVED** [review]
Looks good, suggest @ndevenish do the news but otherwise gently suggest stripping some less than useful comments out, otherwise looks good thank you

**Comment by dagewa** [comment]
Thanks @graeme-winter.  The comments etc. came directly from `FormatCBFMiniEigerMaxIVBio`, which I copied as the start point. Should I also edit that format as part of this PR?

**Comment by graeme-winter** [comment]
> Thanks @graeme-winter. The comments etc. came directly from `FormatCBFMiniEigerMaxIVBio`, which I copied as the start point. Should I also edit that format as part of this PR?

I would say no, because that is out of scope, I just thought avoiding _extra_ useless commentary was worthwhile. Not hostile to taking a look through the wider repo at some point to do some lint removal.

**Pull request merged**